content/en/docs/tasks/administer-cluster/kms-provider.md

```diff
@@ -92,9 +92,12 @@ resources:
    - identity: {}
 ```
 
-2. Set the `--experimental-encryption-provider-config` flag on the kube-apiserver to point to the location of the configuration file.
+2. Set the `--encryption-provider-config` flag on the kube-apiserver to point to the location of the configuration file.
 3. Restart your API server.
 
+Note:
+If you are running on Kubernetes of version lower than 1.13, you need to use `EncryptionConfig` as the kind of the object as well as `--experimental-encryption-provider-config` as the kube-apiserver flag.
```

Comment:
suggestion:
> The alpha version of the encryption feature prior to 1.13 required a config file with `kind: EncryptionConfig` and `apiVersion: v1`, and used the `--experimental-encryption-provider-config` flag.

Reply:
I would include the same note on the encrypt-data.md page

Reply:
Thanks, that sounds much better. Modified in latest push.